reporter/src/main/kotlin/reporters/StaticHtmlReporter.kt

```diff
@@ -124,11 +178,53 @@ class StaticHtmlReporter : Reporter {
                         issueTable(reportTableModel.issueSummary)
                     }
 
+                    h2 {
+                        id = "project-dependencies"
+                        +"Projects and Dependencies"
+                    }
+
+                    details {
```

Comment:
Right below the code calls functions like `projectTable` and `metadataTable`. In order to keep the abstraction level consistent please also move the code for that `details` block into a separate function.

Reply:
Done